tests/NATS.Extensions.Microsoft.DependencyInjection.Tests/MyJsonContext.cs

```diff
@@ -0,0 +1,8 @@
+using System.Text.Json.Serialization;
+
+namespace NATS.Extensions.Microsoft.DependencyInjection.Tests;
+
+[JsonSerializable(typeof(MyData))]
+internal partial class MyJsonContext : JsonSerializerContext;
+
+public record MyData(string Name);
```

Comment:
create a property for this with getter+setter. net6 have problems with source generators with init properties.

Reply:
Done